sdk/iot/Azure.Iot.Hub.Service/src/IfMatchPrecondition.cs

```diff
@@ -8,12 +8,6 @@ namespace Azure.Iot.Hub.Service
     /// </summary>
     public enum IfMatchPrecondition
     {
-        /// <summary>
-        /// Perform this operation regardless of if the provided resource matches the service's representation
-        /// of the object. This will cause the HTTP request to be sent with no ifMatch header. The service will never respond with a 412 error code with this setting.
-        /// </summary>
-        Unconditional,
```

Comment:
this meant that we wouldn't send the ifnatch header in the http request, did this behavior change from the service? do they always require an ifmatch header to be set?

Reply:
Yes, you cannot update/delete an existing entity without providing any if-match headers.